feature-search/src/androidMain/kotlin/search/data/api/SearchStreamService.kt

```diff
@@ -1,8 +1,8 @@
-package com.codandotv.streamplayerapp.feature_list_streams.search.data.api
+package search.data.api
```

Comment:
The same point I said before, I think it is something going wrong with our name space for this module